styles/styles.css

```diff
@@ -391,11 +397,17 @@ main .section.isi {
 }
 
 main .section.isi h2 {
```

Comment:
not sure if this is the right place to be, I think it may be more suitable to fragment.css, because it refers to important safety information section, that is treated in the fragment.css, wdyt?

Reply:
I actually disagree - I think we already have "too much" in the fragments block, IMO there should only be code in there to load the fragment and nothing else. All the styling and code that the fragment itself needs should be provided by the fragment (or global scripts/css in this case, since it is loaded on every page anyway).